src/steam_network/protocol_client.py

```diff
@@ -285,6 +285,8 @@ async def finalize_login(self, username:str, steam_id:int, refresh_token:str, au
             self._auth_lost_handler = auth_lost_handler
         elif result == EResult.AccessDenied:
             return UserActionRequired.InvalidAuthData
+        elif result == EResult.Expired
```

Comment:
missing a `:` here; I hit an invalid syntax issue locally when installing the latest version

Reply:
Yeah, that was dumb. Mb, updated